renderer/plugin.js

```diff
@@ -0,0 +1,67 @@
+// Renderer plugin can't be tested as of now
+// Please move the logic to other classes and test them independently
+// Let the plugin class delegate functionality to these classes
+Plugin.mathtext = {};
+
+Plugin.mathtext.RendererPlugin = Plugin.extend({
+  _type: 'org.ekstep.mathtext',
+  _isContainer: false,
+  _render: true,
+  initPlugin: function(data) {
+    var pluginData;
+
+    if (!_.isUndefined(data.data))
+      pluginData = JSON.parse(data.data.__cdata);
+
+    var pid = data._id || data.id;
+
+    this.id = _.uniqueId('org.ekstep.mathtext');
+    this._data = data;
+    var mathData = _.clone(this._data);
+    mathData.id = pid;
+
+    mathData.__text = _.isUndefined(pluginData.latex) ? mathData.__text : pluginData.latex;
+    var dims = this.relativeDims();
+    var div = document.getElementById(data.id);
+    if (div) {
+      jQuery("#" + data.id).remove();
+    }
+    div = document.createElement('div');
+    if (mathData.style)
+      div.setAttribute("style", mathData.style);
+    div.id = mathData.id;
+    div.classList.add('math-text');
```

Comment:
Use constants

Reply:
Constants cannot be used, because div element has to be created dynamically for each equation.